<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>152</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>K = </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditKA"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMultiplicarKA">
       <property name="text">
        <string>A * K</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonCriarA">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>152</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>P = </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPotenciaB"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPotenciaB">
       <property name="text">
        <string>B ^ P</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>152</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>P = </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPotenciaA"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPotenciaA">
       <property name="text">
        <string>A ^ P</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>101</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditQuantidadeDeColunasA"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Coluna:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditQuantidadeDeLinhasA"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Linha:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonTranspostaB">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Transposta B</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTranspostaA">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Transposta A</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>101</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Linha:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditQuantidadeDeLinhasB"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Coluna:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditQuantidadeDeColunasB"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonCriarB">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditMatrizA">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>70</y>
      <width>381</width>
      <height>271</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Iguais</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Diferentes</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Triangular Superior</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Triangular Inferior</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Matriz Identidade</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Matriz Simetrica</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Matriz Ortogonal</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Matriz de Permutacao</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>A</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>B</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButtonMultiplicacao">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonApagarA">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Apagar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditResultado">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>152</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>K = </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditKB"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMultiplicarKB">
       <property name="text">
        <string>B * K</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonApagarB">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Apagar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSubtracao">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSoma">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditMatrizB">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
